packages/web/src/components/Categories.tsx

```diff
@@ -194,6 +194,10 @@ export const categories: Record<TCategoryName, TCategoryDefinition> = {
       stub: "shelters",
     },
     subCategories: [
+      {
+        text: "Safe Parking",
+        stub: "safe-parking",
+      },
```

Comment:
Mike got back to me about ordering. Let's put this between "Abused" and "Pregnant" sub-categories. After that's done, this is good to go!

Reply:
Copy that. It's done!